apps/v1/consts.go

```diff
@@ -0,0 +1,71 @@
+package v1
+
+const (
+	DeploymentStatusReasonAnnotation  = "openshift.io/deployment.status-reason"
+	DeploymentIgnorePodAnnotation     = "deploy.openshift.io/deployer-pod.ignore"
+	DeploymentPodAnnotation           = "openshift.io/deployer-pod.name"
+	DeployerPodCreatedAtAnnotation    = "openshift.io/deployer-pod.created-at"
+	DeployerPodStartedAtAnnotation    = "openshift.io/deployer-pod.started-at"
+	DeployerPodCompletedAtAnnotation  = "openshift.io/deployer-pod.completed-at"
+	DeploymentReplicasAnnotation      = "openshift.io/deployment.replicas"
+	DesiredReplicasAnnotation         = "kubectl.kubernetes.io/desired-replicas"
```

Comment:
I don't think are API is a good match for this prefix

Reply:
?

Reply:
There's nothing wrong with defining the constant here.  The point of a constant is that you can copy them anywhere, because they have to be constant.